events-api/src/modules/events/events.service.ts

```diff
@@ -1,10 +1,55 @@
 import { EventModel } from "#/modules/events/events.model";
 import { EventsRepository } from "#/modules/events/events.repository";
+import { faker } from "@faker-js/faker";
+import { EventsBodyDataDto } from "./dto/requests/events-body-data.dto";
+import { EventsUpdateDto } from "./dto/requests/events-update-data.dto";
 
 export class EventsService {
   static async getEvents(): Promise<EventModel[]> {
     const events = await EventsRepository.getAll();
 
     return events;
   }
+
+  static async getEventByID(eventId: string){
+    const event = (await this.getEvents()).filter(event => event.eventId === eventId);    
+    return event;
+  }
+
+  static async newEvent(eventData: EventsBodyDataDto): Promise<EventModel> {
+    const newEvent: EventModel = {
+      eventId: faker.string.uuid(),
```

Comment:
Don't use faker for generation of uuids. faker should be used only for tests or mocks